include/odp/api/spec/timer.h

```diff
@@ -75,11 +75,14 @@ int odp_timer_res_capability(odp_timer_clk_src_t clk_src,
  * frequency multiplier and the minimum timeout resolution. If there is no requirement for timeout
  * resolution, it is set to zero.
  *
- * A successful call overwrites all 'capa' fields with the resulting values. Return value
- * is 1 when timer capability meets or exceeds all requested values. The call returns 0, when
- * the requested base frequency is not supported exactly, but capability meets or exceeds all other
- * requested values. In this case, the call overwrites 'base_freq_hz' with the closest supported
- * frequency.
+ * A successful call overwrites 'capa' fields in the following way. The call returns 1, when
+ * timer capability supports the requested base frequency exactly, and meets or exceeds other
```

Comment:
Maybe `... the following ways. The call returns 1, when timer supports ...`

Reply:
Fixed in v2